agent/fw_drupal.c

```diff
@@ -455,8 +468,9 @@ static void nr_drupal_wrap_hook_within_module_invoke_all(
    * available.
    */
 #if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
-     && !defined OVERWRITE_ZEND_EXECUTE_DATA
-  zval* curr_hook = (zval*)nr_stack_get_top(&NRPRG(drupal_module_invoke_all_hooks));
+    && !defined OVERWRITE_ZEND_EXECUTE_DATA
+  zval* curr_hook
+      = (zval*)nr_stack_get_top(&NRPRG(drupal_module_invoke_all_hooks));
```

Comment:
This is just the formatting change, right?

Reply:
yes